### PR TITLE
Add **kwargs to ClusterRoleBinding __init__

### PR DESCRIPTION
**Pull request by jpeimer**
##### Short description:
Add **kwargs to ClusterRoleBinding __init__

### CODE REVIEW

**Comment by jpeimer** [comment]
!verified

File: ocp_resources/cluster_role_binding.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def __init__(
         name=None,
         cluster_role=None,
         subjects=None,
+        client=None,
```
Comment (myakove) [inline ocp_resources/cluster_role_binding.py:19]: Please add **kwargs, and add **kwargs to super() call.

File: ocp_resources/cluster_role_binding.py [hunk 2]
```diff
@@ -16,6 +16,7 @@ def __init__(
...
     ):
         super().__init__(name=name)
```
Comment (myakove) [inline ocp_resources/cluster_role_binding.py:21]: `super().__init__(name=name, **kwargs)`

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by jpeimer** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Comment by jpeimer** [comment]
!verified

**Pull request merged**